File: JavaSource/src/main/java/ru/akhcheck/patterns/command/cookery/Cook.java
package ru.akhcheck.patterns.command.cookery;

import java.util.List;
import java.util.Map;

/** Исполнитель. Про команды не знает ничего и умеет только готовить. */
public class Cook {
    private static final Map<String, List<String>> RECIPES = Map.of(
            "Borsh", List.of("Boil water", "Crop vegetables", "Mix"),
            "Solyanka", List.of("Boil water", "Add meat", "Add mix of meat")
    );

    public void prepare(String dish) {
        List<String> steps = RECIPES.get(dish);
        if (steps == null) {
            System.out.println("Cook does not know how to make " + dish);
            return;
        }

        for (String step : steps) {
            System.out.println("  " + step);
        }
        System.out.println("  " + dish + " is ready");
    }
}

File: Command/python-source/cookery/Cook.py
from __future__ import annotations

from typing import ClassVar


class Cook:
    """Исполнитель. Про команды не знает ничего и умеет только готовить."""

    RECIPES: ClassVar[dict[str, list[str]]] = {
        "Borsh": ["Boil water", "Crop vegetables", "Mix"],
        "Solyanka": ["Boil water", "Add meat", "Add mix of meat"],
    }

    def prepare(self, dish: str) -> None:
        steps = self.RECIPES.get(dish)
        if steps is None:
            print(f"Cook does not know how to make {dish}")
            return

        for step in steps:
            print(f"  {step}")
        print(f"  {dish} is ready")
